node-bb/README.md

```diff
@@ -0,0 +1,168 @@
+and update it per the instructions in [Configuration](#configuration) section. 
+```cp example/config.json /opt/triplea-forums/config.json```  
+2. With the config.json run the triplea-forums:latest image using the command 
+```docker run -d --rm --name triplea-forums -v /opt/triple-forums/config.json:/usr/src/app/config.json -v /opt/triple-forums/uploads:/usr/src/app/public/uploads -p 8080:4567 triplea-forums:latest```
+
+### Stopping
+
+To stop the running container, run the following: ```docker stop triplea-forums```
+
+### MongoDB  
+
+Mongo is the database that backs triplea-forums. You can install and run a local copy by following the
+instructions [here](https://docs.mongodb.com/manual/installation/) or you can run a Docker image with Mongo already
+installed.
+
+Before running the Docker image it is important to know which version of Mongo you want to run. As of this writing the 
+latest version is 4.4. In most cases running that will be fine however if you need a different version, you can find 
+the list of Docker image versions on [https://hub.docker.com/_/mongo](https://hub.docker.com/_/mongo).
+
+#### Mac OSX and Windows
+
+This command will start the Docker container:
+
+```
+docker run --rm --name mongodb -p 27017:27017 -d mongo:4.4
+```
+
+To stop the container, run ```docker stop mongodb```
+
+The data saved will persist between shutdowns because it is being saved in the VirtualBox instance running on your os.
+For more information on that, see the section Where to Store Data at 
+[https://hub.docker.com/_/mongo](https://hub.docker.com/_/mongo).
+
+#### Linux and Unix
+
+While Linux and Unix can use the same command as Mac OSX to start the container, the data stored is not in VirtualBox.
+Instead, it is stored directly on the host at `/var/lib/docker/volumes/{volume hash}/_data`. It's far easier for backup
+purposes if you mount a volume directory for storing the data. The below command uses a volume mount.
+
+This command will start the Docker container:
+
+```
+docker run --rm --name mongodb -p 27017:27017 -v /opt/triple-forums/datadir:/data/db -d mongo:4.4
+```
+
+To stop the container, run ```docker stop mongodb```
+
+The data saved is stored in the volume path you specify with the `-v` flag. In this case `/opt/triple-forums/datadir`.  
+
+### NodeBB
+
+The table below lists the version of NodeBB that TripleA Forums is currently running. This document 
+is written to that version.
+
+| TripleA Forums | NodeBB Version | Package Json                                                                             |
+|----------------|----------------|------------------------------------------------------------------------------------------|
+| Current        | 1.13.3         | [package.json source](https://github.com/NodeBB/NodeBB/blob/v1.13.3/install/package.json)|
+| Future         | 1.14.3         | [package.json source](https://github.com/NodeBB/NodeBB/blob/v1.14.3/install/package.json)|
```

Comment:
Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6342386)